handler/handler.go

```diff
@@ -0,0 +1,80 @@
+package handler
+
+import (
+	"unsafe"
+
+	"github.com/tetratelabs/tinymem"
+)
+
+// HandleFn is the entry-point function which defaults to Next.
+var HandleFn func() = Next
+
+// handle isn't exported to avoid end users invoking it.
```

Comment:
`handle is only exported to the host`?

Can't help but trip seeing the word export just below it

Reply:
lol yeah agree that's confusing